data-model/SDP-sample-records/codicological-unit_sample1.json

```diff
@@ -0,0 +1,143 @@
+{
+  "id": "ark:/21198/a310jfBd",
+  "type": "Codicological Unit",
+  "label": "Part 1",
+  "extent": ["f. 1-152"],
+  "dimensions": [{
+    "type": "average-folio",
+    "value": "320 x 250 x 92 mm"
+  }],
+  "support": {
+    "type": "Parchment",
+    "features": [],
+    "note": ["Quire 20 is paper."]
+  },
+  "condition": {
+    "note": [],
+    "features": []
+  },
+  "writing_and_hands": [{
+    "language": "Syriac",
+    "writing_system": "Syriac",
+    "script": "Estrangela",
+    "ink_color": "black",
+    "hand_note": ["Taw, Alaph, and Dalath/Resh all exhibit features of Serto."],
+    "textual_artifacts": ["ark:/21198/z1x07bdf"],
+    "extent": ["f. 1-152"]
+  }],
+  "page_layout": [{
+    "columns": 1,
+    "number_of_lines": 26,
+    "column_width": "",
+    "margin_width": "",
+    "features": [],
+    "note": [""],
+    "extent": ["f. 1-152"]
+  }],
+  "collation": {
+    "viscodex": "",
+    "formula": ["Pastedown+flyleaf, Quire 1: 1x10 (10), Quire 2: 1x6 (16), Quire 3: 1x8 (24), Quire 4: 1+1x8 (33), Quires 5-12: 8x8 (97), Quire 13: 1x8+1 (106), Quire 14: 1x6+1+1 (114), Quire 15: 1x8 (122), Quire 16: 6x1 (128), Quire 17: 1x6+1 (135), Quire 18: 1x6+1 (142), Quire 19: 1x6 (148), Quire 20 (paper): 1x4 (152)."],
+    "note": ["Quire signatures at beginning and end of most quires."],
+    "features": ["Signatures"]
+  },
+  "paratexts": [{
+      "type": "reader-note",
+      "locus": "f. 114v",
+      "transcription": {
+        "syr": ["ܢܗܘܐ ܕܘܟܪܢܐ ܛܒܐ ܩܕܡ ܐܠܗܐ ܘܡܫܝܚܗ ܠܝܥܩܘܒ ܘܠܒܪ ܫܒܘܢ ܒܢ̈ܘܗܝ ܕܒܣܝܠ ܕܡܢ ܐܝܘ ܩܪܝܬܐ"]
+      },
+      "translation": {
+        "en": ["May there be a blessed rememberance before God and his Christ for Jacob and Bar-Shabbon, sons of Basil, from the village of Yw."]
+      },
+      "note": ["The association between these persons and the scribe, or the production of the manuscript more generally, remains unclear."],
+      "features": []
+    }
+  ],
+  "associated_dates": [
```

Comment:
should “associated_dates” just be “dates” or “date”? “associated” keeps it in line with “associated_persons” (and places) but could be cumbersome. If we treat paratexts like we do persons, places, and dates, then we wouldn’t _have_ to use ‘associated’ necessarily.